create-session/main.c

```diff
@@ -26,20 +26,12 @@ main(int argc, char **argv)
 	path = NULL;
 	user_count = 0;
 	ARGBEGIN {
-	case 'd':
-		path = EARGF(usage());
-		if (!*path)
-			eprintf("path was emtpy\n");
-		break;
 	case 'u':
 		if (user_count >= MAX_USERS)
 			eprintf("max users exceeded (%d max)\n", MAX_USERS);
 		users[user_count++] = EARGF(usage());
 		break;
-	case 'h':
-		usage();
-		exit(0);
-		break;
+	COMMON_ARGS;
```

Comment:
Shouldn't common args be first as can be -h?

Reply:
Order doesn't matter as the arguments are passed in order,  and -h will exit(0) either way

Reply:
I would like to have evaluation of common ones at the top, thinks it will be more visible.